packages/devtools_app/lib/src/shared/status_line.dart

```diff
@@ -35,78 +35,44 @@ class StatusLine extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final textTheme = Theme.of(context).textTheme;
-
-    final List<Widget> children = [];
+    return ValueListenableBuilder<bool>(
+      valueListenable: currentScreen.showIsolateSelector,
+      builder: (context, showIsolateSelector, _) {
+        final textTheme = Theme.of(context).textTheme;
 
-    // Have an area for page specific help (always docked to the left).
-    children.add(
-      Expanded(
-        child: Align(
-          alignment: Alignment.centerLeft,
-          child: buildHelpUrlStatus(context, currentScreen, textTheme),
-        ),
-      ),
-    );
+        final children = <Widget>[];
```

Comment:
let's construct this list directly instead of calling children.add a bunch of times:

```
final children = [
  buildHelpUrlStatus(context, currentScreen, textTheme),
  const BulletSpacer(),
  if (showIsolateSelector)
    ...[
          const IsolateSelector(), 
          const BulletSpacer(),
       ],
   ...
];
```

We can also just build this directly as the children property in row. It actually makes the build method a little more readable.
```
Row(
  children: [ ... ],
)
```

Reply:
Good point! Only change to your suggestion is I've pulled getting the children into a separate method